rom/laghos_rom.hpp

```diff
@@ -65,6 +65,7 @@ struct ROM_Options
     bool staticSVD = false; // true: use StaticSVD
     bool useOffset = false; // if true, sample variables minus initial state as an offset
     bool RHSbasis = false; // if true, use bases for nonlinear RHS terms without mass matrix inverses applied
+    bool SNS = false; // if true, use sns
```

Comment:
We know what SNS is, but maybe for users it should be explained briefly, maybe with a reference to the SNS paper.

Reply:
The arXiv ID is of the SNS paper is added. Once our LaghosROM paper is available online, we can use that as reference.